XRTK.Oculus/Packages/com.xrtk.oculus/package.json

```diff
@@ -15,6 +15,6 @@
   "src": "Assets/XRTK.Oculus",
   "author": "XRTK Team (https://github.com/XRTK)",
   "dependencies": {
-    "com.xrtk.core": "0.1.21"
+    "com.xrtk.core": "0.1.25"
```

Comment:
Don't forget to bump the package release version as well

Reply:
Will double-check, I believe you upped the version in a second PR